client/blocks/login/login-form.jsx

```diff
@@ -30,13 +34,16 @@ export class LoginForm extends Component {
 		redirectTo: PropTypes.string,
 		requestError: PropTypes.object,
 		translate: PropTypes.func.isRequired,
+		isFormDisabled: PropTypes.bool,
```

Comment:
From what I understand, `isFormDisabled` is linked to the redux state whereas `isDisabled` is linked to the component state and is used to disable the component for SSR, until the client is fully loaded.